src/components/structural/WelcomeScreen.js

```diff
@@ -83,16 +144,32 @@ class Welcome extends React.Component {
     }
 
     /**
+     * Helper function to convert the shortcuts to an equivalent DOM elements 
+     * 
+     * @param {array} data 
+     */
+     shortcutHelper = (data) => {
+         let shortcuts = [];
+         data.shortcut.forEach((key,i)=>{
+             shortcuts.push(<kbd>{key}</kbd>);
+             if(i < data.shortcut.length-1){
+                 shortcuts.push(" + ");
+             }
+         });
```

Comment:
if you want to get fancy, this is a good use case for `reduce`.
something like this might work:
```
const shortcuts = data.shortcut.reduce((acc, key, i) => {
    acc.push(<kbd>{key}</kbd>);
    if(i < data.shortcut.length-1) {
        shortcuts.push(" + ");
    }
    return acc;
}, []);
```